### PR TITLE
project after combining react and neutralinojs

### PR DESCRIPTION
**Pull request by sachith-1**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE. -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>NeutralinoJs</title>
+    <link
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
```
Comment (shalithasuranga) [inline app/index.html]: Please don't use remote paths. Could you remove this?

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        document.getElementById("aMem").innerHTML =
+          (data["ram"]["available"] / (1024 * 1024)).toFixed(3) + " GB";
+        document.getElementById("tMem").innerHTML =
+          (data["ram"]["total"] / (1024 * 1024)).toFixed(3) + " GB";
+      },
+      function() {
+        document.getElementById("aMem").innerHTML = "Error While Executing";
+      }
+    );
+    document.getElementById("ram").style.visibility = "visible";
```
Comment (shalithasuranga) [inline src/App.js]: Could you please use react state instead. I mean don't use document.get.....

File: app/index.html (hunk 1 above)
Comment (sachith-1) [reply app/index.html]: Remote paths removed

File: src/App.js (hunk 2 above)
Comment (sachith-1) [reply src/App.js]: docu.getEle replaced with react state

**Review by shalithasuranga — APPROVED** [review]
Nice work :+1:

**Pull request merged**